packages/composites/circuit-breaker/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`execute valid result method should return correct result value with the second source if the first one fail 1`] = `
+Object {
+  "data": Object {
+    "result": 3068.06,
+  },
+  "debug": undefined,
+  "jobRunID": "1",
+  "result": 3068.06,
+  "statusCode": 200,
+}
+`;
+
+exports[`execute valid result method should return correct result values 1`] = `
+Object {
+  "data": Object {
+    "result": 3068.06,
+  },
+  "debug": undefined,
+  "jobRunID": "1",
+  "result": 3068.06,
+  "statusCode": 200,
+}
```

Comment:
Right now, these have the same result value. It would be valueable to test where these are different.

Reply:
done